src/containers/AccountSettings/accountSettings.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Icon } from 'antd';
+
+import Logo from '../../components/LogoHeader';
+import SubHeading from '../../components/subHeading';
+import NavigationBar from '../../components/navigationBar';
+
+import './accountSettings.css';
+
+const accountSettings = ({ info }) => {
+  return (
+    <div className="settings">
+      <Logo />
+
+      <div className="settings__desktop-container">
+        <section className="settings__title">
+          <SubHeading text="Account Settings" />
+
+          <Link
+            to="/confirm-password"
+            className={
+              info.createdAccount
```

Comment:
```suggestion
             info && info.createdAccount
```

Reply:
because    info.createdAccount will give us an undefined error if there is no `info`!